modules/connatixBidAdapter.js

```diff
@@ -31,6 +31,10 @@ const BIDDER_CODE = 'connatix';
 const AD_URL = 'https://capi.connatix.com/rtb/hba';
 const DEFAULT_MAX_TTL = '3600';
 const DEFAULT_CURRENCY = 'USD';
+const CNX_IDS = 'cnx_ids';
```

Comment:
I woudn't name const like this since it's not holding any cnx ids

Reply:
user_ids?

Reply:
CNX_IDS_LOCAL_STORAGE_KEY or create a LOCAL_STORAGE_KEYS object that has the CNX_IDS inside as a prop